src/common/models/measure/measure.ts

```diff
@@ -224,22 +232,34 @@ export class Measure implements Instance<MeasureValue, MeasureJS> {
     return this.formatFn(datum[this.name] as number);
   }
 
-  public change(propertyName: string, newValue: any): Measure {
+  public change(propertyName: string, newValue: any): this {
```

Comment:
Just to be clear this here is a really confusing place to use it, I was just trying it out. see the immutable class PR for a 'real' use case